feat(aws-lambda): adds manual layer instructions (#3052)
getsentry/sentry-docs: src/platforms/node/guides/aws-lambda/layer/index.mdx

---
title: AWS Lambda Layer (Node)
description: "Learn how to set up Sentry with a Layer"
---

> An alternative way of installing Sentry is using a [Lambda Layer](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html). This happens instead of adding `@sentry/serverless` with NPM and Yarn. Your can still import Sentry as usual:

Reviewer: ```suggestion
You can also install Sentry using a [Lambda Layer](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html) instead of adding `@sentry/serverless` with `npm` or `yarn`. Import Sentry as usual:
```

```javascript
const Sentry = require("@sentry/serverless");
```

> <Alert level="warning">
> 
> Note, this approach will not work if you use Webpack.
> 
> </Alert>

Reviewer: Let's clarify this -because it's either a Note or a Warning. Generally, we're moving away from colorful boxes that are more distracting than useful. In addition, I'm not quite clear on what we'd like to say - are we telling people not to use this approach if they're also using Webpack? In that case, perhaps:
```suggestion
<Note>

This installation method will not work if you are using Webpack.

</Note>
```

> To add the Sentry Layer, go to your Lambda function, press **Layers**, then **Add a Layer**.

Reviewer: ```suggestion
Then add the Sentry Layer by navigating to your Lambda function. Select **Layers**, then **Add a Layer**.
```

   ![](lambda_view.png)

> Then go to the **Specify an ARN** tab as shown:

Reviewer: ```suggestion
 **Specify an ARN** tab as illustrated:
```

   ![](add_layer.png)
   

> Then below, set the region and copy the ARN value shown below into the input.

Reviewer: ```suggestion
Finally, set the region and copy the provided ARN value into the input.
```
